webcrypto/aes.go

```diff
@@ -75,6 +102,9 @@ func (akgp *AesKeyGenParams) GenerateKey(
 	return &key, nil
 }
 
+// Ensure that AesKeyGenParams implements the KeyGenerator interface.
+var _ KeyGenerator = &AesKeyGenParams{}
```

Comment:
```suggestion
var _ KeyGenerator = &AESKeyGenParams{}
```
nit: If the name is AES (an acronym) why are we using a lower version?

Reply:
That's a fair point, with which I keep battling since the beginning of the project. The names are not following the convention, but the specification's. In the specs, they do not use the same uppercasing as Go for acronyms. As I'm trying to stay as close to the specs as possible, also to make binding one to the other as easy as possible, I followed the specs, but in other places I actually don't 🤦🏻 

I've opened an issue to track and address this #39